Gemfile

```
@@ -2,7 +2,7 @@ source 'https://rubygems.org'
 
 gem 'fastlane'
 
-gem 'cocoapods', '1.11.3'
+gem 'cocoapods', '>= 1.11.2'
```

@brainbicycle that was coming from the rn diff gemfile from the upgrade site, do you think we should change it to:

```suggestion
gem 'cocoapods', '>= 1.11.3'
```

---

I think it is probably fine 🤷